src/dependencies/ietf-te-topology-state@2017-10-25.yang

```diff
@@ -0,0 +1,296 @@
+module ietf-te-topology-state {
```

Comment:
I am wondering if we should be committing and using a fixed set of dependent YANG modules. The problem is if these dependent modules change, as they have recently, we would be verifying against old models. I would rather that we expand on the script I started (but commented out in validate-and-gen-trees.sh file to download YANG modules on demand. Something I want to discuss as part of YANG doctors lunch in IETF 101.

Reply:
I agree that we should be validating against latest, that's what yangvalidator.com does afaik. To compile the yang files locally ideally the tool should go and get the latest e.g. using xym.